### PR TITLE
LPS-191548 Test

### PR DESCRIPTION
**Pull request by p-albuquerque**

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx [hunk 1]
```diff
@@ -325,32 +321,19 @@ export default function ViewObjectDefinitions({
...
 
-			makeFetch();
-		}
-
-		Liferay.on('addObjectDefinition', () =>
-			setShowModal((previousState: ViewObjectDefinitionsModals) => ({
-				...previousState,
-				addObjectDefinition: true,
-			}))
-		);
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx:349]: Hey @p-albuquerque, this section is responsible for opening the modal to add a new object definition.

**Comment by p-albuquerque** [comment]
ci:test:sf

**Pull request closed without merging**